client/post-editor/editor-sidebar/index.jsx

```diff
@@ -27,8 +29,31 @@ export default class EditorSidebar extends Component {
 		isPrivate: PropTypes.bool,
 	}
 
+	constructor() {
+		super();
+		this.state = { showFeedback: false };
+	}
+
+	openFeedbackPane = () => {
+		this.setState( { showFeedback: true } );
+	};
+	closeFeedbackPane = () => {
+		this.setState( { showFeedback: false } );
+	};
```

Comment:
yeah here we know the return value isn't being used, but also we have to acknowledge that _every_ function in JavaScript returns a value, so even if we use the function body syntax we're still returning `undefined`, which ironically, is a value.